blaze/compute/tests/test_dask_dataframe.py

```diff
@@ -121,13 +121,13 @@ def test_selection_inner_inputs():
 
 
 def test_reductions():
```

Comment:
It isn't: `count, sum, min, max` return integers for both pandas and dask dataframes.  `mean, var, std` return floats here, probably since the exact values happen to be floats.

Dask dataframe results also end up as `numpy.int64` and `numpy.float64`.